frontend/src/Context.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { initialState, reducer } from './reducer';
+
+export const StoreContext = React.createContext({});
+
+export function ContextProvider(props) {
+    const [state, dispatch ] = React.useReducer(reducer, initialState);
+    const context = { state, dispatch };
+
+    return (
+        <StoreContext.Provider value={context}>
+            {props.children}
+        </StoreContext.Provider>
+    )
+}
```

Comment:
Nice, I think Context will work well for the scale of our app

Reply:
I also just realized I'm on chatReader instead of my personal one, but this is AJ btw haha